### ISSUE TITLE
Crashes a 02:15 with panic: reflect: Call using zero Value argument

### ISSUE THREAD

**Issue by ThoreKr** [muety/wakapi#78]
Good evening,
since upgrading to 16.2 and continuing in 17.2 I regularly see these crashes:
```
Dec 11 01:03:27 wakapi[172106]: [100.434ms] [rows:1] INSERT INTO `heartbeats` (`user_id`,`entity`,`type`,`category`,`project`,`branch`,`language`,`is_write`,`editor`,`operating_system`,`machine`,`time`) VALUES ("thore","/home/thore/Projects/backend/functions/tests/unit/recommendation/get_recommendations.spec.ts","file","","backend","feature/recommender","TypeScript",false,"vscode","Linux","nodename","2020-12-11 01:03:26.676")
Dec 11 01:03:27 wakapi[172106]: 127.0.0.1 - - [11/Dec/2020:01:03:27 +0100] "POST /api/heartbeat HTTP/1.1" 201 27
Dec 11 02:15:00 wakapi[172106]: panic: reflect: Call using zero Value argument
Dec 11 02:15:00 wakapi[172106]: goroutine 9 [running]:
Dec 11 02:15:00 wakapi[172106]: reflect.Value.call(0xbbc700, 0xc00009c120, 0x13, 0xc88311, 0x4, 0xc000360000, 0x1, 0x1, 0xc0005d8cb0, 0x454eac, ...)
Dec 11 02:15:00 wakapi[172106]:         /usr/lib/go/src/reflect/value.go:401 +0x1365
Dec 11 02:15:00 wakapi[172106]: reflect.Value.Call(0xbbc700, 0xc00009c120, 0x13, 0xc000360000, 0x1, 0x1, 0x47039b, 0x6c791fa83f32, 0xc02f5aca6d)
Dec 11 02:15:00 wakapi[172106]:         /usr/lib/go/src/reflect/value.go:337 +0xb9
Dec 11 02:15:00 wakapi[172106]: github.com/jasonlvhit/gocron.callJobFuncWithParams(0xbbc700, 0xc00009c120, 0xc00009c130, 0x1, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0)
Dec 11 02:15:00 wakapi[172106]:         /home/thore/go/pkg/mod/github.com/jasonlvhit/gocron@v0.0.0-20191106203602-f82992d443f4/gocron.go:141 +0x21a
Dec 11 02:15:00 wakapi[172106]: github.com/jasonlvhit/gocron.(*Job).run(0xc0004a25a0, 0x0, 0x0, 0x0, 0x0, 0x0)
Dec 11 02:15:00 wakapi[172106]:         /home/thore/go/pkg/mod/github.com/jasonlvhit/gocron@v0.0.0-20191106203602-f82992d443f4/gocron.go:127 +0x24b
Dec 11 02:15:00 wakapi[172106]: github.com/jasonlvhit/gocron.(*Scheduler).RunPending(0xc000258000)
Dec 11 02:15:00 wakapi[172106]:         /home/thore/go/pkg/mod/github.com/jasonlvhit/gocron@v0.0.0-20191106203602-f82992d443f4/gocron.go:530 +0xa5
Dec 11 02:15:00 wakapi[172106]: github.com/jasonlvhit/gocron.(*Scheduler).Start.func1(0xc000094190, 0xc000258000, 0xc0004c4000)
Dec 11 02:15:00 wakapi[172106]:         /home/thore/go/pkg/mod/github.com/jasonlvhit/gocron@v0.0.0-20191106203602-f82992d443f4/gocron.go:619 +0x45
Dec 11 02:15:00 wakapi[172106]: created by github.com/jasonlvhit/gocron.(*Scheduler).Start
Dec 11 02:15:00 wakapi[172106]:         /home/thore/go/pkg/mod/github.com/jasonlvhit/gocron@v0.0.0-20191106203602-f82992d443f4/gocron.go:615 +0x87

```

Unfortunately I haven't been able to figure out why this happens or what triggers it, just - since migrating it into a systemd unit to restart on failure - that it always happens at 02:15:00.

**Comment by muety** [comment]
I'll investigate, thanks!

**Comment by muety** [comment]
Should be fixed, please update to `1.17.3`.

**Comment by ThoreKr** [comment]
Thank you for the quick response.
